README.md

```diff
@@ -1,75 +1,91 @@
 # rems-admin
 
 ## Running only the REMS server project locally
-1.  Clone the REMS repositories from Github:
+
+1.  Clone the REMS repositories from GitHub:
     ```bash
-    git clone https://github.com/mcode/rems-admin.git rems-admin 
+    git clone https://github.com/mcode/rems-admin.git rems-admin
     ```
-2. Run dockerRunner.sh script
+2.  Run dockerRunner.sh script
     ```bash
     npm run start
     ```
 
 ### How To Override Defaults
-The .env file contains the default URI paths, these can be overwritten from the start command as follows:
- `MONGO_URL=http://example.com SERVER_PORT=6000 npm start`
- 
-Following are a list of modifiable paths: 
-
-| URI Name      | Default |
-| ----------- | ----------- |
-| MONGO_URL | `mongodb://rems-user:pass@127.0.0.1:27017` |
-| MONGO_DB_NAME | `remsadmin` |
-| WHITELIST | `http://localhost, http://localhost:3005` |
-| LOGGING_LEVEL | `debug` |
-| PORT | `8090` |
-| RESOURCE_SERVER | `http://localhost:8090` |
-| AUTH_SERVER_URI | `http://localhost:8090` |
-| VSAC_API_KEY | `changeMe` |
-| SMART_ENDPOINT | `http://localhost:3005/launch` |
-| HTTPS_KEY_PATH | `server.key` |
-| HTTPS_CERT_PATH | `server.cert` |
-| USE_HTTPS | `false`|
-
-## Running the Mongo DB instance 
-1. On the first run use the following command to create a docker mongo instance:
-    ```bash
-        docker run --name rems_local_pims_remsadmin_mongo --expose 27017 -p 27017:27017 -e MONGO_INITDB_ROOT_USERNAME='rems-admin-pims-root' -e MONGO_INITDB_ROOT_PASSWORD='rems-admin-pims-password' -v rems_local_pims_remsadmin_mongo:/data/db -v "$(pwd)"/mongo-init.js:/docker-entrypoint-initdb.d/mongo-init.js mongo
-    ```
-    To stop the running container, simply use ctrl + c
 
+The .env file contains the default URI paths, which can be overwritten from the start command as follows:
+a) `REACT_APP_LAUNCH_URL=http://example.com PORT=6000 npm start` or b) by specifying the environment variables and desired values in a `.env.local`.
+
+> **Bug**: Do note that the `SMART_ENDPOINT` environment variable cannot be overwritten in a `.env.local`, it must be done in the `.env`.
+
+Following are a list of modifiable paths:
+
+| URI Name        | Default                                    |
```

Comment:
not necessarily a change we need right now, but this list could use some sort of description of what each variable is for and why you'd change it.

Reply:
should we make a ticket for this (esp if it might be useful for the connectathon)?

Reply:
REMS-590 was created for this.